libs/components/forms/src/lib/modules/radio/radio-group.component.ts

```diff
@@ -151,18 +163,28 @@ export class SkyRadioGroupComponent
   #_tabIndex: number | undefined;
 
   #changeDetector: ChangeDetectorRef;
+  #radioGroupIdSvc: SkyRadioGroupIdService;
   #ngControl: NgControl | undefined;
 
   constructor(
     changeDetector: ChangeDetectorRef,
+    @Host() radioGroupIdSvc: SkyRadioGroupIdService,
```

Comment:
Is there a particular reason we need `@Host` here? Is it possible that without it we'd somehow get an incorrect instance of `SkyRadioGroupIdService`? Since `SkyRadioGroupIdService` is provided by this class, it shouldn't ever look beyond the host component's injector anyway.

Reply:
You're correct, it's redundant. Removed.